### ISSUE TITLE
Web Accessibility First Aid missing footer

### ISSUE THREAD

**Issue by yatil** [w3c/wai-first-aid#2]
_From @nitedog on March 16, 2018 14:39_

https://www.w3.org/WAI/beta/plan/interim-repairs/

_Copied from original issue: w3c/wai-website#13_

**Comment by yatil** [comment]
Footer from @slhenry via mail:

```
<p><strong>Date:</strong> Updated November 2016. First published March 2006.<br>History: Previously titled "Short Term Website Accessibility Improvements" and "Improving the Accessibility of Your Website".</p>
<p><strong>Editor: </strong><a href="https://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Previous editors: <a href="https://www.w3.org/People/kevin">Kevin White</a> and <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.  Previous contributors: Sharron Rush, Anna Belle Leiserson, Judy Brewer, and <a href="https://www.w3.org/WAI/EO/participants">EOWG Participants</a>.</p>
<p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). First developed with support from <a href="https://www.w3.org/WAI/TIES/"><acronym title="Web Accessibility Initiative: Training, Implementation, Education, Support">WAI-TIES</acronym> Project</a>, then updated with support of the <a href="https://www.w3.org/WAI/ACT/"><acronym title="Web Accessibility Initiative - Cooperation Framework for Guidance on Advanced Technologies, Evaluation Methodologies, and Research Agenda Setting to Support eAccessibility">WAI-ACT</acronym> Project</a>, and later the <a href="https://www.w3.org/WAI/DEV/"><acronym>WAI-DEV</acronym> Project</a>.</p>
```
